README.md

```diff
@@ -24,6 +24,7 @@ cp .env.example .env
 
 ### Install and run application for development
 ```shell
+nvm use
```

Comment:
I think it would be best to separate the NVM commands into their own shell if the users decide not to use NVM. 

Also I think the following would be helpful much like the GOVUK Flask Frontend commands:

```
nvm install
nvm use
node -v
```